### PR TITLE
Feature/next

### PR DESCRIPTION
**Pull request by recover758126**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {createElement, Component} from 'rax';
+import { createElement, Component,createRef } from 'rax';
```
Comment (yacheng) [inline src/index.js]: import { createElement, Component, createRef } from 'rax';

File: README.md [hunk 2]
```diff
@@ -1,15 +1,59 @@
-# rax-barcode [![npm](https://img.shields.io/npm/v/rax-barcode.svg)](https://www.npmjs.com/package/rax-barcode)
+[![npm](https://img.shields.io/npm/v/rax-barcode.svg)](https://www.npmjs.com/package/rax-barcode)
 
-
-
-## Install
+**描述：**
+Barcode 用于显示条形码，基于canvas实现
```
Comment (yacheng) [inline README.md]: Barcode 用于显示条形码，基于 Canvas 实现

Comment (recover758126) [reply README.md]: done

File: src/index.js (hunk 1 above)
Comment (recover758126) [reply src/index.js]: done

**Pull request merged**